README.md

```diff
@@ -180,5 +180,6 @@ In addition to the filter parameters, there is a set of query parameters that ha
 | `eager=[children, parent.movies]` | Which relations to fetch eagerly for the result models. An objection.js relation expression.             |
 | `orderBy=firstName`               | Sort the result by certain property.                                                                     |
 | `orderByDesc=firstName`           | Sort the result by certain property in descending order.                                                 |
+| `orderByAsc=group\|lastName`      | Sort the result by more than one property in ascending order.                                            |
```

Comment:
This seems not very RESTful. Can you adjust the param structure to follow either
`?foo=bar&foo=qux`
or
`?foo=bar,qux`
, as suggested in https://medium.com/raml-api/arrays-in-query-params-33189628fa68